cloudfront-invalidate/requirements.txt

```diff
@@ -0,0 +1,2 @@
+awscli==1.16.218
```

Comment:
Could you update readme how manually installing dependencies of this lambda code? or installing the dependencies of this lambda in npm `init` script?

Reply:
Installation is not needed, because they are bundled in AWS Lambda Python environment.